docs/userguide/torch_forecasting_models.md

```diff
@@ -25,6 +25,7 @@ We assume that you already know about covariates in Darts. If you're new to the
     - [Callbacks](#callbacks)
       - [Early Stopping](#example-with-early-stopping)
       - [Custom Callback](#example-of-custom-callback-to-store-losses)
+      - [MLFlow: train, track and monitor](#example-with-mlflow-autologging)
```

Comment:
I think that the indentation is off, can you shift it to the left once so that it's at the same level as "Callbacks"?

Reply:
Hello @madtoinou ,

I fixed this via commit https://github.com/unit8co/darts/pull/2092/commits/b839c67ce2f970b23d3d9b41e9b4a0ab4ae35315